internal/web3ext/web3ext.go

```diff
@@ -495,6 +495,10 @@ web3._extend({
 			name: 'isRunning',
 			call: 'validator_isRunning'
 		}),
+		new web3._extend.Method({
+			name: 'isValidating',
```

Comment:
What's the usecase of `validator.isRunning` vs `validator.isValidating`? As a user of `validator`, when should I use either one? They sound similar, like if a validator object is running for me it means it's validating, that's why it's called validator... no?

Reply:
Afaik validator can be in 2 states: running and validating.
If validator is 'running' it doesn't mean that it is 'validating'. But if it's validating, it's running also.
Maybe we should remove isRunning?

Reply:
I don't know. I don't understand why a validator would be running and not validating. But if that's a thing, then it's alright.

Reply:
I think running means that is trying to start validating. the user says start, but hasn't yet joined the validators

Reply:
I see. Thanks @jmatosp

Reply:
but, not easy to follow :|